recipes/trinity/build.sh

```diff
@@ -20,6 +20,7 @@ cd $SRC_DIR
 # The compilers aren't propogated across makefiles
 ln -s ${CC} ${PREFIX}/bin/gcc
 ln -s ${CXX} ${PREFIX}/bin/g++
```

Comment:
This isn't always a good idea. The problem is that some tools look for other tools in the directory in which they are themselves.

Reply:
I know, but there are a bunch of Makefiles and, as I wrote, they would all have to be patched since the compiler information isn't sent from one to the other. It looks like I have to do that anyway to get this to work on OSX though, so I guess it's time to bite the bullet.

Reply:
If you are going to do patches:

```
conda debug recipes/trinity
# don't switch, just source the env file
cd some-workdir
git init 
wget xyz
tar xf xyz
git commit -am 'orig'
# edit & build
git commit  -am "what I had to change"
# reiterate
git format patch ... 
```

(I'm using emacs/magit for this, so I can mess with commits up the tree and create clean patches, but it's nice, because you see what you've done to get it to build)